scripts/topiary-analyze-cohort

```diff
@@ -54,65 +54,77 @@ from .mutation_report import print_mutation_report
 parser = argparse.ArgumentParser()
 
 input_group = parser.add_mutually_exclusive_group(required=True)
-input_group.add_argument("--variant-input-dir",
-   type=str,
-   help="Directory containing MAF or VCF input files")
+input_group.add_argument(
```

Comment:
That's an old script that I wrote for Kipp Akers' analysis of TCGA, I'm a little hesitant to try and understand it again. I was just reformatting the whitespace. I think eventually this should either get deleted or "modernized" to use the commandline args from the main Topiary script.